recipes/giddy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+
+{% set name = "giddy" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c284856579ab95ff6a63b6a7c838ab0a48c8669745ea16f8428a012abedb750
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >= 3
+    - pip
+  run:
+    - python >= 3
```

Comment:
Same here:
```suggestion
    - python >=3
```

Reply:
And the others need a space adding to match this pattern. (Building can work but other parts of conda-build which are used for making the feedstock will fail if its not formatted correctly)

Reply:
cool, thank you! should be fixed now